nvdaHelper/remote/winword.cpp

```diff
@@ -304,6 +304,12 @@ int generateHeadingXML(IDispatch* pDispatchParagraph, IDispatch* pDispatchParagr
 	int headingLevel=getHeadingLevelFromParagraph(pDispatchParagraph);
 	if(!headingLevel) return 0;
 	XMLStream<<L"<control role=\"heading\" level=\""<<headingLevel<<L"\" ";
+	// Expose the collapsed state of the heading
+	bool isCollapsed=false;
```

Comment:
Beware to the case!

I think that "bool" lowercase is on 8 bits, whereas "BOOL" uppercase is on 32 bits, hence the crash. Strange that you did not get it.

```suggestion
	BOOL isCollapsed=false;
```